paasta_tools/cli/cmds/push_to_registry.py

```diff
@@ -79,72 +78,105 @@ def add_subparser(subparsers):
         "-f",
         "--force",
         help=(
-            "Do not check if the image is already in the PaaSTA docker registry. "
+            "Do not check if the image is already in the PaaSTA docker registries. "
             "Push it anyway."
         ),
         action="store_true",
     )
+    list_parser.add_argument(
+        "-r",
+        "--registries",
+        help=(
+            "Docker Registry URLs, such as docker-paasta-norcal.yelpcorp.com:443, separated by semicolons."
```

Comment:
can you use nargs for this, rather than semi colon delimeters?
https://docs.python.org/3.6/library/argparse.html#nargs

Reply:
Done.